src/Symfony/Bundle/DebugBundle/Resources/config/services.xml

```diff
@@ -50,24 +51,23 @@
             </call>
         </service>
 
-        <service id="var_dumper.server_dumper" class="Symfony\Component\VarDumper\Dumper\ServerDumper">
-            <argument>null</argument> <!-- server host -->
-            <argument type="service" id="var_dumper.cli_dumper" />
+        <service id="var_dumper.server_connection" class="Symfony\Component\VarDumper\Server\Connection">
```

Comment:
should it be hidden ?

Reply:
Right now, we hide only services with "random" names, not sure this PR should be the one changing this.

Reply:
I don't think that's true. I'm quite sure I reviewed some PRs hiding some service names. But I'm still fine if the main rule is to hide only "random" names.

Reply:
I didn't find any with a quick `grep`, let's let it as is.